lldb/unittests/Core/TelemetryTest.cpp

```diff
@@ -95,3 +107,34 @@ TELEMETRY_TEST(TelemetryTest, PluginTest) {
 
   ASSERT_EQ("FakeTelemetryPlugin", ins->GetInstanceName());
 }
+
+TELEMETRY_TEST(TelemetryTest, ScopedDispatcherTest) {
+  lldb_private::FakePlugin::Initialize();
+  auto *ins = TelemetryManager::GetInstance();
+  ASSERT_NE(ins, nullptr);
+  std::vector<::llvm::telemetry::TelemetryInfo *> expected_entries;
```

Comment:
```suggestion
  std::vector<::llvm::telemetry::TelemetryInfo *> received_entries;
```
(expected it the thing you expect to receive)

Reply:
done